app/assets/stylesheets/static_pages_home.scss

```diff
@@ -0,0 +1,41 @@
+.static_pages_home {
+	height: 100vh; width: 100vw;
+	background: #F0F0F0;
+
+	.row {
+		position: relative; top: 45%;
+		transform: translateY(-50%);
+		h2 {
+			font-family: 'Arvo', sans-serif;
```

Comment:
It is generally a good practice to haev all these as variables defined in on theme.scss file.